pkg/controller/template/render_test.go

```diff
@@ -54,6 +54,10 @@ func TestCloudProvider(t *testing.T) {
 		platform:    configv1.OpenStackPlatformType,
 		featureGate: newFeatures("cluster", "CustomNoUpgrade", []string{cloudprovider.ExternalCloudProviderFeature}, nil),
 		res:         "external",
+	}, {
+		platform:    configv1.PowerVSPlatformType,
```

Comment:
Any background on this change? This was added a few months ago from @Prashanth684 who said it would only be external once there was a CCM for PowerVS, did that happen (I'm unsure about the status)

See: https://github.com/openshift/machine-config-operator/pull/2801#discussion_r737709249

Reply:
correct. this should not be external yet and the comment referenced above still holds. could this change be reverted please.

Reply:
I don't have the full context to make a recommendation here, the unit test will break since that information seems to be coming from library-go added in this [commit](https://github.com/openshift/library-go/commit/893d6f91df5b7c1af9284ec508faa3f37bf46fc0)

Reply:
hmm..yeah..i'm checking with the IBM folks but looks like it has been marked as an external provider and i do see changes in CCM too, so this should be good. i'll confirm once i hear back from them.

Reply:
alright yeah so CCM now includes PowerVS: https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/129 which is supported through the IBM cloud provider, so this change is good! thanks @kikisdeliveryservice for notifying and thanks @eggfoobar for this change!